slirp4netns.1.md

```diff
@@ -224,7 +238,7 @@ If `guest_addr` is not specified, then it will be set to the default address tha
 ```console
 (namespace)$ json='{"execute": "list_hostfwd"}'
 (namespace)$ echo -n $json | nc -U /tmp/slirp4netns.sock
-{"return": {"entries": [{"id": 42, "proto": "tcp", "host_addr": "0.0.0.0", "host_port": 8080, "guest_addr": "10.0.2.100", "guest_port": 80}]}}
+{"return": {"entries": [{"id": 42, "proto": "tcp", "host_addr": "0.0.0.0", "host_addr6": "::", "host_port": 8080, "guest_addr": "10.0.2.100", "guest_addr6": "fd00::100", "guest_port": 80}]}}
```

Comment:
I’m not sure we should mix up v4 and v6 in a single entry.

Reply:
What should we return in `api_handle_req_add_hostfwd` when we add both tcp4 and tcp6? An array of IDs?

Reply:
Avoid adding tcp4 and tcp6 in a single request/response.

Mixing up them in a single request/response makes it hard to implement RootlessKit integration.
https://github.com/rootless-containers/rootlesskit/blob/c5481c1cfd6f6c42c8ec1868fbc543f2b0a916a5/pkg/api/openapi.yaml#L70-L90

Reply:
I tried to implement it like Go's net listen, which does listen on both 4 and 6 if you set tcp with global ip 0.0.0.0 or :: if I understood it correctly.

If we don't need that I can change it like that?

- host_addr: 0.0.0.0, proto: tcp -> v4
- host_addr: 0.0.0.0, proto: tcp6 -> error
- host_addr: ::, proto: tcp -> v6
- host_addr: ::, proto: tcp4 -> error